CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ For more information about changelogs, check
 [Keep a Changelog](http://keepachangelog.com) and
 [Vandamme](http://tech-angels.github.io/vandamme).
 
+## 0.2.14 - 2017/01/24
+* [ENHANCEMENT] Add the Funky::Page API to fetch username and name
+  fields from the Facebook Graph API given a page ID
```

Comment:
@bruce-fs If new API is added to the Funky gem, it should be documented in README.md.

That is the place where currently all the public methods of Funky are described.